pom.xml

```diff
@@ -9,7 +9,7 @@
   <packaging>jar</packaging>
 
   <name>RabbitMQ Java Client</name>
-  <description>RabbitMQ Java client</description>
+  <description>The RabbitMQ Java client library allows Java code to interface to AMQP servers.</description>
```

Comment:
I have just copied the description from here: https://www.rabbitmq.com/java-client.html. But no problem, I will replace "AMQP servers" with "RabbitMQ". The description in the pom gets used as "Bundle-Description" in the OSGi manifest file and things look better with a proper outline.